src/workspaces/NewWorkspaceModal/NewWorkspaceModal.ts

```diff
@@ -411,6 +419,73 @@ const NewWorkspaceModal = withDisplayName(
         : 'You need a billing project to create a new workspace.';
     };
 
+    const endingNotice = renderNotice ? renderNotice({ selectedBillingProject }) : undefined;
+
+    const displayAzurePolicyAndWorkspaceInfo =
+      isAzureBillingProject() || (!!cloneWorkspace && isAzureWorkspace(cloneWorkspace));
+    const renderAzurePolicyAndWorkspaceInfo = () => {
+      const renderLink = (href: string, text: string) => {
+        return h(
+          Link,
+          {
+            href,
+            ...Utils.newTabLinkProps,
+            style: { display: 'inline-block', marginTop: '0.25rem' },
+          },
+          [text, icon('pop-out', { size: 14, style: { marginLeft: '0.1rem' } })]
+        );
+      };
```

Comment:
Could extract this into a component?

```suggestion
      const NewWorkspaceModalLink = (props: LinkProps) => {
        const { children, ...otherProps } = props;
        return h(
          Link,
          {
            ...Utils.newTabLinkProps,
            style: { display: 'inline-block', marginTop: '0.25rem' },
            ...otherProps
          },
          [children, icon('pop-out', { size: 14, style: { marginLeft: '0.1rem' } })]
        );
      };
```

Reply:
Done. We probably should generalize it a tad and put it in components… we have this link w/popout thing all over the palce.